CMakeLists.txt

```diff
@@ -75,6 +81,142 @@ target_compile_definitions(notcurses
     FORTIFY_SOURCE=2 _GNU_SOURCE SOURCE_DEFAULT
 )
 
+# libnotcurses++
+set(NCPP_SOURCES
+  src/libcpp/NotCurses.cc
+  src/libcpp/PanelReel.cc
+  src/libcpp/Plane.cc
+  src/libcpp/Root.cc
+  src/libcpp/Tablet.cc
+  src/libcpp/Visual.cc
+  )
+
+add_library(
+  notcurses++
+  SHARED
+  ${NCPP_SOURCES}
+  )
+
+add_library(
+  notcurses++-static
+  STATIC
+  ${NCPP_SOURCES}
+  )
+set_target_properties(
```

Comment:
huh, i thought cmake gave you a static version for free, but apparently not. lame.

Reply:
Yep ;)